docs/source/faq(developers).md

```diff
@@ -258,3 +258,13 @@ rm -rf bower_components
 npm install
 bower install
 ```
+
+#### Q. While trying to build EvalAI from the master branch and run the command docker-compose up:
+
+```
+ERROR: Service 'celery' failed to build: pull access denied for evalai_django, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
+```
+
+Please make sure to clone EvalAI in its default directory with name evalai. This happens because the parent directory changes the name of docker images.
+For instance, the image evalai_django gets renamed to evalai_dev_django if your directory is renamed to EvalAI_dev. 
+Refer to [Markdown Guide](https://github.com/Cloud-CV/EvalAI/pull/2746).
```

Comment:
Please add the reference link from where you found this solution and not of this PR.